packages/testsuite/cypress/support/containers-utils.ts

```diff
@@ -22,9 +42,49 @@ declare global {
     interface Chainable {
       /**
        * Start a Wildfly container. This typical command which needs to executed in before method in every TC.
```

Comment:
> This typical command which needs to executed in before method in every TC.

This command is executed in `before` method in most of the test cases/specifications.

Reply:
updated